src/example-types/number.js

```diff
@@ -20,31 +20,75 @@ module.exports = {
       }),
     },
   }),
-  async result({ field }, search) {
-    let result = await search({
+  async result({ field, min, max }, search) {
+    let rangeFilter = {
```

Comment:
Since this is pretty much identical to the filter function, you should factor out the range filter building logic so its a bit more DRY. A function just at the top of this file is fine